manifests/cli.pp

```diff
@@ -64,6 +64,15 @@
     ' '
   )
 
+  if $::jenkins::cli_env_var {
+    $cmd_environment = [
+          "JENKINS_USER_ID=${::jenkins::cli_username}",
```

Comment:
Please use consistent indenting and remove the leading colons from variables. This is a repeated thing in the PR so I'll only comment it once :)
```suggestion
      "JENKINS_USER_ID=${jenkins::cli_username}",
```

Reply:
fixed the indents using puppet-lint